campaigns/campaigns-model.js

```diff
@@ -0,0 +1,51 @@
+const db = require('../data/config');
+
+function get() {
+  return db('campaigns as c')
+    .join('organizations as o', 'c.org_id', 'o.id')
+    .select('c.*', 'o.name as org_name');
```

Comment:
It's not super important here, but shorthand `c.*` to select all from campaign is computationally slower than explicitly writing it out from the server side.  We are talking about millionths of a second for an app this size, so not important for build week.